### ISSUE TITLE
Vertical marker not moving

### ISSUE THREAD

**Issue by maartenvandillen** [ptomasroos/react-native-multi-slider#257]
- [x] I have searched [existing issues](https://github.com/ptomasroos/react-native-multi-slider/issues)
- [x] I am using the [latest multi slider version](https://www.npmjs.com/package/@ptomasroos/react-native-multi-slider)

On Android the vertical marker is not responding to touches. iOS works fine. I wanted to create a repo that reproduces this but instead I went for another package that does work as expected (@sharcoux/slider). Just to let you know this isn't working for me I decided to create this issue anyway.

## Steps to Reproduce
use the MultiSlider component with prop vertical

## Expected Behavior
Marker responds to touches

## Actual Behavior
Marker remains static and does not move when operated on with finger.

**Comment by maartenvandillen** [comment]
Back to this package, since the other one has problems capturing the touch input. Slider value goes all over the place when finger leaves the component view.

Could you please check if the vertical slider is working for you on Android?

**Comment by jcaviedesc** [comment]
Same problem, any solution?

**Comment by maartenvandillen** [comment]
Yes, I switched to Slider component from react-native-elements package

**Comment by Gjoshi3107** [comment]
@maartenvandillen 
same issue here, could you please help how to resolve this.
using library:-
@cychiuae/react-native-multi-slider
which is npm package containing version 2.2.5

**Comment by Gjoshi3107** [comment]
working corectly on "@ptomasroos/react-native-multi-slider": "^2.2.2"
